md5_test.go

```diff
@@ -0,0 +1,32 @@
+package hashutil
+
+import (
+	"crypto/md5"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const emptyMd5 = "d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
don't understand why is this called empty, maybe testMd5?

Reply:
its hex representation of `zero-length hash`